frontend/src/components/VSearchResultsGrid/VAudioResult.vue

```diff
@@ -30,8 +31,11 @@ const props = withDefaults(
 const { $sendCustomEvent } = useNuxtApp()
 const searchStore = useSearchStore()
 
-const { audio } = toRefs(props)
-const { isHidden: shouldBlur } = useSensitiveMedia(audio)
+const { isHidden: shouldBlur } = useSensitiveMedia(ref(props.audio))
+
+const href = computed(() => {
```

Comment:
```suggestion
const audioLink = computed(() => {
```

Reply:
I prefer to have the same property names in both `VAudioResult` and `VImageResult` components. We could change it to `link`, but I don't think it makes that much of a difference.